lib/tasks/ahoy_activity_rollup.rake

```diff
@@ -0,0 +1,79 @@
+namespace :fromthepage do
+  desc "Daily Task to Rollup Ahoy Events into Transcribe minutes by Date, User, and Collection"
```

Comment:
Because we're only going to be running this task once a day, we need some slightly helpful indication in the UI that you can't include today's date.  

The defaults include today's date and probably shouldn't.

Reply:
This has been updated and at least partially addressed. There are some complications re: localization, but we're close.